timescaledb/tutorials/nfl-fantasy-league.md

```diff
@@ -174,19 +174,249 @@ conn.close()
+
+### **Number of plays vs points scored**
+
+We will be using the following query to get data on the number of plays and final score for each game during the 2018 season. This data will be separated by team so that we can compare the number of plays with a team's win or loss.
+
+```sql
+with play_count as (
+-- Count distinct plays, join on the stadium and game tables for team names and game date
+select gameid, count(playdescription) as plays, si.team_name, g.game_date 
+from play p 
+left join stadium_info si on p.possessionteam = si.team_abbreviation 
+left join game g on p.gameid = g.game_id 
+group by gameid, si.team_name, game_date
+), visiting_games as (
+-- Join on scores to grab only the visting team's data
+select gameid, plays, s.visitor_team as team_name, s.visitor_score as team_score from play_count p
+inner join scores s on p.team_name like '%' || s.visitor_team || '%' 
+and p.game_date = s."date"
+), home_games as (
+-- Join on scores to grab only the home team's data
+select gameid, plays, s.home_team as team_name , s.home_score as team_score from play_count p
+inner join scores s on p.team_name like '%' || s.home_team || '%' 
+and p.game_date = s."date"
+)
+-- union the two resulting tables together
+select * from visiting_games
+union all
+select * from home_games
+order by gameid asc, team_score desc
+```
+With the python script below, we will use our query above to create a scatterplot displaying each team's outcome for every game in the season. The scatterplot will be grouped, showing the winning team's plays and scores as gold, and the losing team's plays and scores as brown. 
+
+```python
+# Import needed python modules
+import psycopg2
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+
+# Define the connection string, you will have to update this string for your database
+CONNECTION = "dbname=tsdb user=tsdbadmin password={PASSWORD} host={HOST} port={PORT} sslmode=require"
+
+# Define the Postgres query 
+query = 'with play_count as ( '\
+    'select gameid, count(playdescription) as plays, si.team_name, g.game_date  '\
+    'from play p  '\
+    'left join stadium_info si on p.possessionteam = si.team_abbreviation  '\
+    'left join game g on p.gameid = g.game_id  '\
+    'group by gameid, si.team_name, game_date '\
+    '), visiting_games as ( '\
+    'select gameid, plays, s.visitor_team as team_name, s.visitor_score as team_score from play_count p '\
+    'inner join scores s on p.team_name like \'%\' || s.visitor_team || \'%\'  '\
+    'and p.game_date = s."date" '\
+    '), home_games as ( '\
+    'select gameid, plays, s.home_team as team_name , s.home_score as team_score from play_count p '\
+    'inner join scores s on p.team_name like \'%\' || s.home_team || \'%\'  '\
+    'and p.game_date = s."date" '\
+    ') '\
+    'select * from visiting_games '\
+    'union all '\
+    'select * from home_games '\
+    'order by gameid asc, team_score desc'
```

Comment:
In other places in docs, triple quotes were used """ very long string """  for multi-line strings, maybe update it here as well to be consistent